src/EditorFeatures/CSharp/SplitStringLiteral/SplitStringLiteralOptions.cs

```diff
@@ -8,5 +8,5 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.SplitStringLiteral;
 
 internal sealed class SplitStringLiteralOptions
 {
-    public static Option2<bool> Enabled = new("SplitStringLiteralOptions_Enabled", defaultValue: true);
+    public static Option2<bool> Enabled = new("csharp_split_string_literal_options_enabled", defaultValue: true);
```

Comment:
We don't want a literal translation of the previous name to `_`-style name, but rather a better name for what the option actually does. For this one, something like `csharp_split_string_literals_on_enter` would be appropriate, as that is how we describe it in Tools > Options.

<img width="139" alt="image" src="https://user-images.githubusercontent.com/41759/219488841-d3fcd3f5-f78e-4487-ba41-5a354e4c8458.png">

Reply:
I wonder if we should change this to "... on return" in both UI and option name though, since we use "... on return" elsewhere, so we should be consistent:

<img width="212" alt="image" src="https://user-images.githubusercontent.com/41759/219489255-7f49b1e4-dca6-438d-8996-fd1b9c50f45d.png">

Reply:
We can use `_on_return` in the config name now and leave rename of the UI resource to another PR.

Reply:
Understood, let me revisit all the names here